.github/workflows/plugins_react_tests.yml

```
@@ -28,7 +31,7 @@ jobs:
         uses: theforeman/gha-matrix-builder@v0
 
   test:
-    name: ${{ matrix.plugin }} with Ruby ${{ matrix.ruby }} and Node ${{ matrix.node }}
+    name: ${{ matrix.plugin.repo }} with Ruby ${{ matrix.ruby }} and Node ${{ matrix.node }}
```

```suggestion
    name: ${{ matrix.plugin.name }} with Ruby ${{ matrix.ruby }} and Node ${{ matrix.node }}
```

Given we have the name now, I think this is nicer